docs/user-authentication.txt

```diff
@@ -154,6 +155,107 @@ in the Laravel Sanctum guide.
    To learn more about the ``DocumentModel`` trait, see
    :ref:`laravel-third-party-model` in the Eloquent Model Class guide.
 
+.. _laravel-user-auth-passport:
+
+Laravel Passport
+~~~~~~~~~~~~~~~~
+
+Laravel Passport is an OAuth 2.0 server implementation that offers
+API authentication for Laravel applications. Use Laravel Passport if
+your application requires OAuth2 support.
+
+Install Laravel Passport
+````````````````````````
+
+To install Laravel Passport and run the database migrations required
+to store OAuth2 clients, run the following command from your project root:
+
+.. code-block:: bash
+
+   php artisan install:api --passport
+
+Next, navigate to your ``User`` model and add the ``Laravel\Passport\HasApiTokens``
+trait. This trait provides helper methods that allow you to inspect a user's
+authentication token and scopes. The following code shows how to add ``Laravel\Passport\HasApiTokens``
+to your ``app\Models\User.php`` file:
+
+.. code-block:: php
+
+   <?php
+
+   namespace App\Models;
+
+   use MongoDB\Laravel\Auth\User as Authenticatable;
+   use Laravel\Passport\HasApiTokens;
+
+   class User extends Authenticatable
+   {
+      use HasApiTokens;
+      ...
+   }
+
+Then, define an ``api`` authentication guard in your ``config\auth.php``
+file and set the ``driver`` option to ``passport``. This instructs your
+application to use Laravel Passport's ``TokenGuard`` class to authenticate
+API requests. The following example adds the ``api`` authentication guard
+to the ``guards`` array:
+
+.. code-block:: php
+   :emphasize-lines: 7-10
+
+   'guards' => [
+      'web' => [
+         'driver' => 'session',
+         'provider' => 'users',
+      ],
```

Comment:
Is this other entry default?

Reply:
Yes the web entry is added automatically